app/views/posts/show.html.erb

```diff
@@ -1,25 +1,55 @@
-<div class='container'>
-  <div class='row card mt-2 mb-2 m-0 pt-2'>
-    <h5 class='col-md-12'>
+<div class="container">
+  <div class="post-details card mt-2 mb-2 pt-2">
+    <h5 class="post-title">
       <%= link_to "#{@post.title} by #{@post.author.name}", user_post_path(@user, @post) %>
     </h5>
-    <div class='col-md-12 pt-2 text-end'>
-      <label class='me-2'>
-        Comments: <span><%= @post.comments_counter %></span>
-      </label>
-      <label>
-        Likes: <span><%= @post.likes_counter %></span>
-      </label>
+    <div class="post-metadata row text-end">
+      <div class="col-md-6">
+        <label class="me-2">Comments: <span id="comments_counter"><%= @post.comments_counter %></span></label>
+      </div>
+      <div class="col-md-6">
+        <label>Likes: <span><%= @post.likes_counter %></span></label>
+      </div>
```

Comment:
Great job so far in your implementation of the forms on your blog app. However, I noticed a few issues that when adjusted would help improve the user experience and functionalities of the app.
  -  Kindly consider implementing the like functionality such that when a user click on the like button a like gets created.